.github/workflows/enum-auto-updater.yml

```diff
@@ -25,11 +25,6 @@ jobs:
       - name: Install dependencies
         run: cd tools/@aws-cdk/enum-updater && yarn install --frozen-lockfile && yarn build
 
-      - name: Update enum static mapping
-        run: |
-          cd tools/@aws-cdk/enum-updater
-          ./bin/update-static-enum-mapping
-
       - name: Check for changes
```

Comment:
this step needs to move the other workflow, so we will only create a PR if there is changes to the static mapping file, or we do not need to push any prs

Reply:
actually, it is already in the other flow, so it should be removed from this one

Reply:
I ment this section `static-mapping-check`